tests/screenshot.spec.ts

```diff
@@ -107,16 +107,112 @@ test('browser_take_screenshot (outputDir)', async ({ startClient }, testInfo) =>
     },
   })).toContainTextContent(`Navigate to data:text/html`);
 
-  await client.callTool({
+  expect(await client.callTool({
     name: 'browser_take_screenshot',
     arguments: {},
+  })).toEqual({
+    content: [
+      {
+        data: expect.any(String),
+        mimeType: 'image/jpeg',
+        type: 'image',
+      },
+      {
+        text: expect.stringMatching(
+            new RegExp(`// Screenshot viewport and save it as ${testInfo.outputPath('output')}/page-\\d{4}-\\d{2}-\\d{2}T\\d{2}-\\d{2}-\\d{2}\\-\\d{3}Z\\.\\jpeg`)
+        ),
+        type: 'text',
+      },
+    ],
   });
 
+  const files = [...fs.readdirSync(outputDir)];
+
   expect(fs.existsSync(outputDir)).toBeTruthy();
-  expect([...fs.readdirSync(outputDir)]).toHaveLength(1);
+  expect(files).toHaveLength(1);
+  expect(files[0]).toMatch(/^page-\d{4}-\d{2}-\d{2}T\d{2}-\d{2}-\d{2}-\d{3}Z\.jpeg$/);
 });
 
-test('browser_take_screenshot (omitBase64)', async ({ startClient }) => {
+test('browser_take_screenshot (raw: true)', async ({ startClient }, testInfo) => {
```

Comment:
These are very similar, you can generate them via for (const raw of [undefined, true])